py-bindings/ompl/base/__init__.py

```diff
@@ -1,2 +1 @@
-from ompl import util
```

Comment:
This change "avoids" the major issue on Mac I mention in the PR description, but doesn't solve it if the user imports `ompl.util` or other packages themselves.

I didn't notice any negative side effects from removing this.